galaxy_ng/app/settings.py

```diff
@@ -91,3 +93,5 @@
 
 X_PULP_CONTENT_HOST = "localhost"
 X_PULP_CONTENT_PORT = 24816
+
+os.environ['SDB_NOTIFY_HOST'] = os.popen('ip route').read().split(' ')[2]
```

Comment:
Worth pointing out, I think this only works if you're running Docker on Linux hosts, not "Docker for Mac".

Reply:
Is there a workaround? I think we have some folks developing on Mac OS.

Reply:
Yep @chouseknecht there is. I can include that. I _think_ it's only needed for the OLDDDD docker for mac that ran in a VM and not the newer docker for mac that runs in a hypervisor.